swagger/v1/api_v1.json

```diff
@@ -103,45 +103,45 @@
                 "example": [
                   {
                     "type_of": "article",
-                    "id": 5137,
-                    "title": "The Yellow Meads of Asphodel168",
-                    "description": "Scenester tofu cold-pressed flannel vhs. Yr park portland organic offal fingerstache forage....",
-                    "readable_publish_date": "Aug 30",
-                    "slug": "the-yellow-meads-of-asphodel168-3bgf",
-                    "path": "/username372/the-yellow-meads-of-asphodel168-3bgf",
-                    "url": "http://localhost:3000/username372/the-yellow-meads-of-asphodel168-3bgf",
+                    "id": 574,
+                    "title": "The Waste Land1",
+                    "description": "Irony franzen forage post-ironic. Chia cold-pressed brunch banh mi you probably haven't heard of them...",
+                    "readable_publish_date": "Sep 1",
+                    "slug": "the-waste-land1-4mei",
+                    "path": "/username1/the-waste-land1-4mei",
+                    "url": "http://localhost:3000/username1/the-waste-land1-4mei",
```

Comment:
TIL each time we bump the OpenAPI file we get a bunch of updates because these are generated via factory models (randomly generated on each run) 🙂